src/Database/Model.php

```diff
@@ -149,12 +149,15 @@ class Model extends EloquentModel
      */
     protected static $relationTypes = ['hasOne', 'hasMany', 'belongsTo', 'belongsToMany', 'morphTo', 'morphOne', 'morphMany', 'morphToMany', 'morphedByMany', 'attachOne', 'attachMany', 'hasManyThrough'];
 
+    protected static $eventsBooted = [];
+
     /**
      * Constructor
      */
     public function __construct(array $attributes = [])
     {
         parent::__construct();
+        static::bootNicerEvents();
```

Comment:
No real reason for this to be static anymore! Yay! I have revised this to be a local method.